app/test_main.py

```diff
@@ -1,3 +1,21 @@
+import pytest
 from app.main import get_human_age
 
-# write your code here
+
+@pytest.mark.parametrize(
+    "value1,value2,result",
+    [
+        (-3, -3, [0, 0]),
+        (10, 10, [0, 0]),
+        (16, 16, [1, 1]),
+        (24, 24, [2, 2]),
+        (100, 100, [21, 17]),
+        (32, 94, [4, 16])
+    ])
```

Comment:
In case there are no arguments on the same line with an opening parenthesis, your closing parenthesis should not have any arguments on the same line as well. Please, resolve this issue